modules/ROOT/pages/external-configuration.adoc

```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
 
 == MicroProfile Config
 
-xref:reference:feature/mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
+xref:mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
 With MicroProfile Config, you can inject the external configuration into services in the containers without repackaging them.
 
-Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:reference:config/server-configuration-overview.adoc[system properties, system environment variables], properties files, XML files, or data sources.
+Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:/docs/21.0.0.2/reference/config/server-configuration-overview.html[system properties, system environment variables], properties files, XML files, or data sources.
```

Comment:
to link to the server config overview, use a relative link:

`xref:reference:config/server-configuration-overview.adoc`